pyproject.toml

```diff
@@ -109,6 +109,7 @@ disallow_untyped_defs = true
 testpaths = [
     "tests"
 ]
+asyncio_mode = "auto"
```

Comment:
The E2E tests use Playwright, and that requires those tests to be written with async (`async def`). When these tests are managed by pytest and not Django, then pytest needs [`pytest-asyncio`](https://pytest-asyncio.readthedocs.io/en/latest/concepts.html) to handle those async tests.

And by default , `pytest-asyncio` expects us to explicitly manually set which tests are meant to be async and which not, by tagging the async functions with their decorator.

This setting configures `pytest-asyncio` to automatically identify async tests by whether they are written with `async def` or plain `def`.